src/components/DataTable/DataTable.js

```diff
@@ -1,24 +1,23 @@
 import React, { PropTypes, Component } from 'react';
-
-import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table';
-import parseBootstrapTableData from './util';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
 
 var DataTable = React.createClass({
 
-    render: function () {
-
-        var tableData = parseBootstrapTableData(this.props.data);
+    propTypes: {
+        data: React.PropTypes.array
+    },
 
+    render: function () {
         return (
             <div id={'table'}>
-                <BootstrapTable data={tableData} striped={true} hover={true} >
-                    <TableHeaderColumn dataField="time" isKey={true} dataAlign="center" dataSort={true}>Model Period</TableHeaderColumn>
+                <BootstrapTable data={this.props.data} striped={true} hover={true} >
+                    <TableHeaderColumn dataField="model_period" isKey={true} dataAlign="center" dataSort={true}>Model Period</TableHeaderColumn>
                     <TableHeaderColumn dataField="run" dataAlign="center" dataSort={true}>Run</TableHeaderColumn>
                     <TableHeaderColumn dataField="min" dataAlign="center" dataSort={true}>Min</TableHeaderColumn>
                     <TableHeaderColumn dataField="max" dataAlign="center" dataSort={true}>Max</TableHeaderColumn>
-                    <TableHeaderColumn dataField="mean" dataAlign="center" dataSort={true}>W.Mean</TableHeaderColumn>
+                    <TableHeaderColumn dataField="w_mean" dataAlign="center" dataSort={true}>W.Mean</TableHeaderColumn>
```

Comment:
What's the "w" stand for in the "w_mean" and "w_stdev"? Weighted? What are the weights?